python/hsfs/feature_store.py

```diff
@@ -979,6 +1083,16 @@ def get_or_create_feature_view(
         """Get feature view metadata object or create a new one if it doesn't exist. This method doesn't update
         existing feature view metadata object.
 
+        !!! example
+            ```python
+            feature_view = fs.get_or_create_feature_view(
+                name='bitcoin_feature_view',
+                version=1,
+                # transformation_functions=transformation_functions,
+                # query=query
+            )
+            ```
```

Comment:
`get_or_create_feature_view` requires `query=query`. uncomment and don't include commented lines in snippet